src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObj = {};
+
+  sourceString.split(';').forEach((elem) => {
+    const startValueIndex = elem.indexOf(':');
+
+    const parametr = elem.slice(0, startValueIndex).trim();
+    const value = elem.slice(startValueIndex + 1).trim();
+
+    parametr.replace(/\n/g, '');
+
+    if (parametr) {
+      convertedObj[parametr] = value;
```

Comment:
You are correctly checking if the `parametr` variable is not an empty string before adding it to the `convertedObj`. However, you are not checking if the `value` variable is not an empty string. There can be cases where there is a style parameter without a value.